tests/testutils.py

```diff
@@ -78,20 +78,20 @@ def _tear_down_database(self, dbname):
             password=POSTGRES_PASSWORD,
             host=POSTGRES_HOST,
         )
+        conn.autocommit = True
         cur = conn.cursor()
         cur.execute("DROP DATABASE %s;" % (dbname,))
-        cur.commit()
         cur.close()
         conn.close()
 
     def setUp(self):
         reactor = ExtendedMemoryReactorClock()
 
         config = {"apps": {}, "log": {"setup": {"version": 1}}}
-        config = merge_left_with_defaults(CONFIG_DEFAULTS, config)
 
         self.config_setup(config)
```

Comment:
Not sure if you want this line as it looks like it'll try and add `dbfile` regardless of whether you're using postgres or sqlite?

Reply:
Are you sure? I can't see that?

Reply:
https://github.com/matrix-org/sygnal/blob/b784e46f3a22700efac8560d0ab0230e7c3f955f/tests/testutils.py#L34 ?

Reply:
Ah, I was using an old checkout. This method looks fine on master :)